xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -193,7 +193,7 @@ void CPVRChannelGroups::RemoveFromAllGroups(const std::vector<std::shared_ptr<CP
 {
   for (const auto& channel : channelsToRemove)
   {
-    /* remove this channel from all non-system groups */
+    //remove this channel from all non-system groups.
```

Comment:
I'd keep the space for legibility but that's relly just nit-picking now :-)

Reply:
Ooops, this is just a typo. Thanks.